pom.xml

```diff
@@ -399,17 +399,15 @@
                     </executions>
                 </plugin>
                 <plugin>
-                    <groupId>org.sonatype.plugins</groupId>
-                    <artifactId>nexus-staging-maven-plugin</artifactId>
-                    <version>${nexus-staging-maven-plugin.version}</version>
+                    <groupId>org.sonatype.central</groupId>
+                    <artifactId>central-publishing-maven-plugin</artifactId>
+                    <version>${central-publishing-maven-plugin.version}</version>
                     <extensions>true</extensions>
                     <configuration>
-                        <serverId>ossrh</serverId>
-                        <nexusUrl>https://oss.sonatype.org/</nexusUrl>
-                        <autoReleaseAfterClose>true</autoReleaseAfterClose>
-                        <stagingProgressTimeoutMinutes>15</stagingProgressTimeoutMinutes>
+                      <publishingServerId>central-sonatype-com</publishingServerId>
```

Comment:
![image](https://github.com/user-attachments/assets/b8857d70-a290-47e8-941e-e0c15c67c8de)

Currently the "release.settings.xml" still contains the ossrh server:

![image](https://github.com/user-attachments/assets/22b341b3-7519-4745-a2f6-143209cdcd95)

I would expect this PR to complete, except for the tokens set in the github organization

Reply:
Happy with the changes? I already updated the secrets in our ogranisation